### DISCUSSION TITLE
Cannot login to my app with discord OAuth2 using cypress

### DISCUSSION THREAD

**Discussion by BugrahanDurukan** [cypress-io/cypress#17234] [Questions and Help]
Hello, I recently came across cypress. I really liked it at the first sight. Cypress has helped me a lot in terms of speed. However, there are a couple of issues that came with it. My app uses discord OAuth2 to log in. When I use cypress with chrome to log in to my app, my app throws me to the login back again. My app doesn't do that when I use it manually or simply use mocha. I really like cypress and I wanna recommend everyone in my organization to use it but I really need to fix this issue first. 

"Login" button clicked on the main page >> app sends me to discord OAuth2 authentication page >> "authenticate" button clicked >> redirect back to main page??????
Do I need to do something similar to the documentation provided for the Google OAuth2

Also, When I use cypress with electron it logs in correctly but the requests I send to my backend would return 401

![image](https://user-images.githubusercontent.com/34229203/124763693-ed779100-df3c-11eb-9cae-d33ac1c5059b.png)

**Comment by davifsroberto** [comment]
Hello Friend!

I have the same problem, but I still haven't solved it...

Does your backend require a cookie?

Access manually and see if accessing the URL generates an automatic cookie for you.

Then analyze the same thing using cypress.
